docs_app/content/guide/operators.md

```diff
@@ -0,0 +1,141 @@
+# Operators
+
+An operator is just a function that takes an Observable as its input and returns another Observable.  For example, there is an operator called `map()`, that is equivalent to the Array method of the same name. Just as `[1, 2, 3].map(x => x * x)` will yield `[1, 4, 9]`, the Observable created like this:
```

Comment:
could you add links to to the api docs of the mentioned operators

Reply:
Done